cli/command/container/opts.go

```diff
@@ -899,3 +921,12 @@ func validateAttach(val string) (string, error) {
 	}
 	return val, errors.Errorf("valid streams are STDIN, STDOUT and STDERR")
 }
+
+func convertToNetworkmode(netmode opts.NetworkOpt) container.NetworkMode {
```

Comment:
It would be useful but opts.NetworkOpt is shared between services and container (docker run) and involves different conversion methods.